### ISSUE TITLE
Bug in `SlevomatCodingStandard.Namespaces.FullyQualifiedGlobalConstants`

### ISSUE THREAD

**Issue by asispts** [slevomat/coding-standard#1621]
I found a bug when using these rules:
```xml
<rule ref="SlevomatCodingStandard.Namespaces.FullyQualifiedGlobalConstants" />
<rule ref="SlevomatCodingStandard.Namespaces.ReferenceUsedNamesOnly">
    <properties>
        <property name="searchAnnotations" value="true" />
        <property name="allowFullyQualifiedGlobalConstants" value="true" />
    </properties>
</rule>
```

In the following code, `User` and `Role` will be formatted to `\User` and `\Role` (See image below).
```php
<?php

namespace App;

use App\Entity\{ User, Role as EntityRole};

final class Foobar
{
    /**
     * @var User
     */
    private $user;

    /**
     * @var EntityRole
     */
    private $role;
}
```

![001](https://github.com/slevomat/coding-standard/assets/79239132/2049f78d-37bb-44c0-98ae-7dd7f7213539)

**:information_source:  NOTE:**
- I already use `SlevomatCodingStandard.Namespaces.DisallowGroupUse` to prevent group use.

**Comment by kukulich** [comment]
Yes, we don't support group uses.

**Comment by asispts** [comment]
I see. I don't support group uses as well.

However, the rule shouldn't fix a code in the wrong place either. There's no need to format or fix constants at the namespace declaration.

Please, check this report.
```
  5 | ERROR | [ ] Type User should not be used, but referenced via a fully qualified name. (SlevomatCodingStandard.Namespaces.UseOnlyWhitelistedNamespaces.NonFullyQualified)
  5 | ERROR | [ ] Group use declaration is disallowed, use single use for every import. (SlevomatCodingStandard.Namespaces.DisallowGroupUse.DisallowedGroupUse)
  5 | ERROR | [x] Constant User should be referenced via a fully qualified name. (SlevomatCodingStandard.Namespaces.FullyQualifiedGlobalConstants.NonFullyQualified)
  5 | ERROR | [ ] Multiple used types per use statement are forbidden. (SlevomatCodingStandard.Namespaces.MultipleUsesPerLine.MultipleUsesPerLine)
  5 | ERROR | [x] Constant Role should be referenced via a fully qualified name. (SlevomatCodingStandard.Namespaces.FullyQualifiedGlobalConstants.NonFullyQualified)
```

**Comment by kukulich** [comment]
> the rule shouldn't fix a code in the wrong place either

Yes, it shouldn't but somebody has to send PR with the fix :) We will not do it :)

**Comment by asispts** [comment]
Thanks. I'll see if I can fix it :)

**Comment by kukulich** [comment]
The fix can be really easy like https://github.com/slevomat/coding-standard/pull/1555
